src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -117,6 +117,12 @@
     <EmbeddedResource Include="MetadataTests\Invalid\ManyMethodSpecs.vb" />
     <EmbeddedResource Include="MetadataTests\Invalid\Obfuscated.dll" />
     <EmbeddedResource Include="MetadataTests\Invalid\Obfuscated2.dll" />
+    <Content Include="MetadataTests\Invalid\Signatures\build.bat" />
+    <Content Include="MetadataTests\Invalid\Signatures\munge.csx" />
+    <Content Include="MetadataTests\Invalid\Signatures\SignatureCycle2.il" />
+    <Content Include="MetadataTests\Invalid\Signatures\TypeSpecInWrongPlace.il" />
+    <EmbeddedResource Include="MetadataTests\Invalid\Signatures\SignatureCycle2.exe" />
```

Comment:
BTW, as soon as we'll have a metadata writer it will be much easier to write these tests without checking in binaries. But for now this is what we do.